roles/custom/matrix_playbook_migration/tasks/validate_config.yml

```diff
@@ -223,6 +223,20 @@
           We found usage of the following variables: {{ matrix_playbook_migration_jitsi_migration_vars.keys() | join(', ') }}
       when: "matrix_playbook_migration_jitsi_migration_vars | length > 0"
 
+- when: matrix_playbook_migration_matrix_jitsi_migration_validation_enabled | bool
```

Comment:
This `when` variable should be adjusted

Reply:
Done → https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/3698/commits/67205297ba40af3d8be471f157872a52e47b8818